src/librustc_session/config.rs

```diff
@@ -70,6 +70,16 @@ impl FromStr for Sanitizer {
     }
 }
 
+#[derive(Clone, Hash, Debug)]
+pub enum CFGuard {
+    /// Emit Control Flow Guard metadata but no checks.
+    Nochecks,
```

Comment:
Please rename this variant to `MetadataOnly` or something else along the lines. `Nochecks` seems to imply CFG is disabled entirely, which it isn’t.

Reply:
The `nochecks` variant was intended to match the corresponding `-cfguard,nochecks` option in Clang (https://reviews.llvm.org/rL339420). Should we keep this terminology for consistency?

The intention is really to convey that this is "Control Flow Guard without any run-time checks". Is there somewhere in the documentation I could explain this?

Reply:
I guess you can keep it in that case. Just `s/Nochecks/NoChecks/`. There are a couple of places for this documentation to go, including in the manpages and the [unstable book](https://doc.rust-lang.org/stable/unstable-book/compiler-flags.html).